init.js

```diff
@@ -51,8 +51,8 @@ require.cache[SHPath].exports = class WebSocketShard extends SH {
 	identify() {
 		let hotReload = this.manager.client.options.hotReload;
 		if(hotReload) {
-			const data = hotReload.sessionData?[this.id] || this.manager.client._loadSessions(this.id);
-			if(data?.id && !this.sessionID) {
+			const data = (hotReload.sessionData || this.manager.client._loadSessions(this.id))?.[this.id]
```

Comment:
loadSessions was changed to return the session object directly if an id is given, so it needs to either be changed back to give an object of sessions, or change this to reflect that

Reply:
Well I thought it would be better to not have data be potentially two different objects. Users would supply sessionData as an object with keys of shardID and values of that shards session. Whereas _loadSessions with an id present returns just the session object meaning data could be different values. Makes more sense to assume they both return the same object